mbuild/packing.py

```diff
@@ -86,13 +93,16 @@ def fill_box(compound, n_compounds=None, box=None, density=None, overlap=0.2,
     aspect_ratio : list of float
         If a non-cubic box is desired, the ratio of box lengths in the x, y,
         and z directions.
+    fix_orientation : bool or list of bools
+        Specify if compounds should be rotated when filling box,
```

Comment:
Nitpick, but I think this wording is a bit confusing as this makes it sound as if a value of `True` would mean that compounds should be rotated when filling the box (when the behavior is, in fact, opposite). A slight phrase change to "Specify that compounds should not be rotated when filling the box" might clear this up.